pkg/cvo/cvo_scenarios_test.go

```diff
@@ -2554,12 +2554,12 @@ func TestCVO_ParallelError(t *testing.T) {
 	}
 	expectGet(t, actions[0], "clusterversions", "", "version")
 
-	// check the worker status is initially set to reconciling
-	if status := worker.Status(); status.Reconciling || status.Completed != 0 {
-		t.Fatalf("The worker should be reconciling from the beginning: %#v", status)
+	// check the worker status is initially set to initializing
+	if status := worker.Status(); status.State != payload.InitializingPayload || status.Completed != 0 {
+		t.Fatalf("The worker should be initializing from the beginning: %#v", status)
 	}
 	if worker.work.State != payload.InitializingPayload {
-		t.Fatalf("The worker should be reconciling: %v", worker.work)
+		t.Fatalf("The worker should be initializing: %v", worker.work)
```

Comment:
This makes sense that it should initially be initializing but how did the test case pass before? Maybe I didn't see where the test case functionality was changed?